apps/admin/src/components/commons/selectField/SelectField.tsx

```diff
@@ -9,11 +9,12 @@ export function SelectField(props: SelectFieldProps) {
     const handleChange = (e: SelectChangeEvent) => {
         setValue(e.target!.value as string);
     };
+
     return (
-        <Box sx={{ minWidth: 120 }}>
-            <FormControl sx={{ minWidth: 120 }}>
+        <Box>
+            <FormControl sx={{ width: 210 }}>
```

Comment:
C'est dangereux de fixer la width comme cela.
Non bloquant pour le moment mais devra être revisité

Reply:
ok c'est noté!